Software/vortex_ws/src/vision/package.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>vision</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="you@example.com">fatma</maintainer>
+  <license>TODO: License declaration</license>
```

Comment:
@fatma-mohamed-98  please fill all the TODOs as we discussed  yesterday
mail: info@vortex-co.com
license use the same as the repo license